packages/neuron-wallet/package.json

```diff
@@ -42,13 +42,13 @@
     "@iarna/toml": "2.2.5",
     "@ledgerhq/hw-transport-node-hid": "6.27.13",
     "@nervina-labs/ckb-indexer": "0.1.1",
+    "@nervina-labs/neuron-shared": "0.110.1",
```

Comment:
Should be updated along with https://github.com/nervosnetwork/neuron/pull/2702/files#diff-059a0f39aaa673028ecc51dfab39fbdf84452a7b5a830c6820008f82331576acR2